lib/perl/Genome/Model/ClinSeq/Command/CreateMutationSpectrum-exome.t

```diff
@@ -14,27 +14,35 @@ BEGIN {
 
 use above "Genome";
 use Test::More tests => 11;  #One per 'ok', 'is', etc. statement below
-use Genome::Model::ClinSeq::Command::CreateMutationSpectrum;
 use Data::Dumper;
+use Genome::Model::ClinSeq::TestData;
+use Genome::Utility::Test;
+use Sub::Override;
 
-use_ok('Genome::Model::ClinSeq::Command::CreateMutationSpectrum') or die;
+my $pkg = 'Genome::Model::ClinSeq::Command::CreateMutationSpectrum';
+use_ok($pkg);
+
+use Genome::Model::Build::ReferenceSequence;
+my $override = Sub::Override->new(
+    'Genome::Model::Build::ReferenceSequence::full_consensus_path',
+    sub { return '/gscmnt/ams1102/info/model_data/2869585698/build106942997/all_sequences.fa'; }
```

Comment:
I'm not sure the final form of disk virtualization, but I would be more wary of hardcoded paths for now and lean towards lookups as suggested.  It would reduce the amount of possible changes needed in the future.